WPMedia/arc-themes-components
# TMEDIA-696 - Add Image component

Added Image component, tests, scss hooks, and documentation

**Note**: This is just a basic component for now to unblock upcoming work - resizer implementation for images will be done at a later date


### Ticket 
[TMEDIA-696](https://arcpublishing.atlassian.net/browse/TMEDIA-696)

### Testing
* Checkout branch - `git checkout TMEDIA-696-image-component`
* Run storybook - `npm run storybook`
* Check storybook output examples and documentation
* Check chromatic build
